openedx/core/djangoapps/credit/exceptions.py

```diff
@@ -1,42 +1,57 @@
 """Exceptions raised by the credit API. """
 
 
-class InvalidCreditRequirements(Exception):
+class CreditApiBadRequest(Exception):
+    """
+    Could not complete a request to the credit API because
+    the parameters were not valid.
```

Comment:
Minor: This docstring mentions parameters, however not all of the children of `CreditApiBadRequest` deal with parameters.